app/controllers/api/v1/accounts_controller.rb

```diff
@@ -15,6 +15,18 @@ def verify_credentials
     render action: :show
   end
 
+  def update_credentials
+    @account = current_user.account
+
+    account_params = params.permit(:display_name, :note)
```

Comment:
I haven't personally worked on an API that allowed you to update something like an avatar or header, since file uploads can be tricky with REST APIs.